app/features-json/view_models/view_model.rb

```diff
@@ -48,29 +93,46 @@ def included(base)
     module ClassMethods
       # Required method to be called overriden by the created ViewModel
       # in order to indicate which class is the Model from which we are creating the ViewModel form.
-      # @param [Class]
-      def base_model(model)
-        @base_model = model
+      # @param [Class] models
+      def base_models(*models)
+        @base_models = models
       end
 
       # An array of the included attributes of the base_model to be included in the ViewModel.
       # @return [Array<Symbol>] the included attributes for the given ViewModel. Defaults to all attributes.
       def included_attributes
-        return @base_model.instance_variables
+        attributes_per_model = {}
+        @base_models.each do |model|
+          attributes_per_model[model.to_s] = model::ATTRS
+                                             .reject { |method| method.to_s.end_with?("=") }
+                                             .map { |method| "@#{method}".to_sym }
+        end
+        return attributes_per_model
       end
 
       # Creates the ViewModel (in shape of a Hash) from a given object of a certain class type.
       # Defined by the base_model method.
-      # @params [Any] object
-      # @return [Hash] object dictionary representation for the ViewModel.
-      def viewmodel_from!(object:)
-        raise "Override base_model(model) in order to use the ViewModel mixin." if @base_model.nil?
-        raise "Incorrect object type. Expected #{@base_model}, got #{object.class}" unless object.kind_of?(@base_model)
+      # @params [Any] objects
+      # @return [Hash<String, Hash>] object dictionary representation for the ViewModel, where the key
+      # is the name of the class and the value is object dictionary representation.
+      def viewmodel_from!(*objects)
+        viewmodel = {}
+        objects.each do |object|
+          viewmodel[object.class.to_s] = _viewmodel_from(object)
+        end
+        return viewmodel
+      end
+
+      private
+
+      def _viewmodel_from(object)
```

Comment:
This is a bit of a nitpick / preference thing, but it's not really convention to prefix private methods with underscore `_` in Ruby (like you would in Python). Would it be possible to rename this method to `viewmodel_form`?

Reply:
It has the same name as the public method, and I want to make sure it's just the same method but I'm private way to allow better code management😕